src/pp-annotator/editor/EditorContent.tsx

```diff
@@ -158,35 +159,30 @@ export default class EditorContent extends Component<IEditorContentProps, IEdito
       <div className={classNames('pp-widget', styles.self)}>
         <div className="pp-editor-head-bar">
           <label className="priority-header"> Co dodajesz? </label>
-          <Popup
-            on="click"
-            hideOnScroll={true}
-            trigger={<div className="priority-help"><i className="help circle icon"/></div>}
-            flowing={true}
-            hoverable={true}
-          >
-            {/*TODO just an instruction stub*/}
-            <Grid centered={true} divided={true} columns={3}>
-              <Grid.Column textAlign="center">
-                <Header as="h4">Niebieski przypis</Header>
-              </Grid.Column>
-              <Grid.Column textAlign="center">
-                <Header as="h4">Żółty przypis</Header>
-              </Grid.Column>
-              <Grid.Column textAlign="center">
-                <Header as="h4">Pomarańczowy przypis</Header>
-              </Grid.Column>
-            </Grid>
-          </Popup>
           <div className={styles.headerButtons}>
-            <PriorityButton type={AnnotationPriorities.NORMAL} onClick={this.setPriority} priority={priority}>
-              dodatkowa informacja
+            <PriorityButton
+              type={AnnotationPriorities.NORMAL}
+              onClick={this.setPriority}
+              priority={priority}
+              tooltipText="Przypis nie jest niezbędny, ale może być użyteczny"
+            >
+              {annotationPrioritiesLabels.NORMAL}
             </PriorityButton>
-            <PriorityButton type={AnnotationPriorities.WARNING} onClick={this.setPriority} priority={priority}>
-              wyjaśnienie
+            <PriorityButton
+              type={AnnotationPriorities.WARNING}
+              onClick={this.setPriority}
+              priority={priority}
+              tooltipText="Bez tego przypisu czytelnik może być wprowadzony w&nbsp;błąd"
```

Comment:
why the `&nbsp;` ?

Reply:
To prevent a line break (just found out you can use &nbsp; for this purpose)